src/en/koharu/src/eu/kanade/tachiyomi/extension/en/koharu/KoharuDto.kt

```diff
@@ -66,7 +66,7 @@ class DataKey(
     fun readableSize() = when {
         size >= 300 * 1024 * 1024 -> "${"%.2f".format(size / (1024.0 * 1024.0 * 1024.0))} GB"
         size >= 100 * 1024 -> "${"%.2f".format(size / (1024.0 * 1024.0))} MB"
-        size >= 1024 -> "${"%.2f".format(size / (1024.0))} KB"
+        size >= 1024 -> "${"%.2f".format(size / (1024.0))} kB"

```

Comment:
Using `kB` here is wrong. Should be `KB` according to https://en.wikipedia.org/wiki/Kilobyte#Definitions_and_usage

>  [...] one kilobyte is 1000 bytes. The unit symbol is kB.
>
> The binary meaning of the kilobyte for 1024 bytes typically uses the symbol KB, with an uppercase letter _K_. 

You should be following the latter standard, since you're working with 1024

(strictly it should be KiB, but you already rejected that ¯\\_(ツ)\_\/¯)

Reply:
Sorry if it seemed like I rejected your idea, that was not my intention. I just forgot to upload the dto file 😅.